change/office-ui-fabric-react-2020-05-27-20-15-46-peoplepicker-duplicates.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Fix examples to properly remove duplicates",
```

It's good to include the component name in this message since it's included in release notes.
```suggestion
  "comment": "PeoplePicker: Fix examples to properly remove duplicates",
```